src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+  },
+  'urlmatcher-null': {
+    action: 'Nếu bạn sử dụng `UrlMatcher`, kiểu dữ liệu đã được cập nhật để có thể luôn luôn trả về `null`.',
+  },
+  'v10-more-details': {
+    action:
+      'Để nắm thêm thông tin chi tiết về các vấn đề bị khai tử, các thay đổi code tự động, và các thay đổi khác, tham khảo [guide angular.io](https://angular.io/guide/updating-to-version-10)'
+  },
+  'universal-baseurl': {
+    action: 'Đối với những người dùng Angular Universal, nếu bạn dùng `useAbsoluteUrl` để thiết lập `platform-server`, từ bây giờ bạn cần phải xác định thêm cả `baseUrl`.',
+  },
+  'v11 ng update': {
+    action: 'Thực thi lệnh `ng update @angular/core @angular/cli` để nâng cấp lên phiên bản 11 cho Angular.'
+  },
+  'v11 versions': {
+    action: 'Angular từ giờ cần phải có [TypeScript 4.0](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/). `ng update` sẽ thực hiện thay đổi này cho bạn.'
+  },
+  'v11 browser support': {
+    action: 'Hỗ trợ cho trình duyệt IE9, IE10, và IE Mobile đã được gỡ bỏ. Điều này đã được thông báo ở [v10 update](http://blog.angular.io/version-10-of-angular-now-available-78960babd41#c357).'
+  },
+  'webpack5 optin': {
+    action: 'Bạn có thể đăng ký tham gia sử dụng Webpack 5 bằng cách sử dụng Yarn (thay vì NPM) và thêm thuộc tính  `"resolutions": {"webpack": "^5.0.0"}` vào `package.json` của bạn.'
+  },
+  'ng new strict prompt': {
+    action: 'Khi khởi tạo dự án mới bằng Angular CLI, bạn sẽ được hỏi là có muốn bật kiểu nghiêm ngặt (strict mode) hay không. Việc này sẽ cấu hình TypeScript và trình biên dịch Angular cho việc kiểm tra kiểu dữ liệu nghiêm ngặt hơn, và sẽ áp dụng ngân sách cho độ lớn của các gói tài nguyên nhỏ hơn. Bạn có thể dùng flag `--strict=true` hoặc `--strict-false` để bỏ qua câu hỏi này từ Angular CLI.'
+  },
+  'v11 router relativeLinkResolution': {
+    action: 'Nếu bạn sử dụng Router, giá trị mặc định của `relativeLinkResolution` từ giờ đã được thay đổi từ `legacy` sang `corrected`. Nếu như ứng dụng của bạn sử dụng giá trị mặc định bằng việc không chỉ định giá trị trong `ExtraOptions` và kích hoạt phiên chuyển trang bằng Relative Links từ những Component con của các Route được khai báo bằng path trống (`path: ""`), bạn cần phải cập nhật cấu hình của `RouterModule` để chỉ định giá trị `legacy` cho `relativeLinkResolution`. Xem [the documentation](https://v11.angular.io/api/router/ExtraOptions#relativeLinkResolution) để biết thêm chi tiết.'
+  },
+  'router initialNavigation': {
+    action: 'Trong Angular Router, những tuỳ chọn bị khai tử trong phiên bản 4 dành cho `initialNavigation` thì đã được gỡ bỏ hoàn toàn ở phiên bản này. Nếu như ứng dụng của bạn có dùng `enabled` hoặc `true` trước đây, bây giờ hãy dùng `enabledNonBlocking` hoặc `enabledBlocking`. Nếu như ứng dụng của bạn có dùng `false` hoặc `legacy_disabled`, giờ hãy sử dụng `disabled`.'
```

Comment:
imo, in this "thì đã được gỡ bỏ", "thì" is unnecessary

Reply:
changed to "Trong Angular Router, những tuỳ chọn không còn được hỗ trợ từ phiên bản 4 dành cho `initialNavigation` đã được gỡ bỏ hoàn toàn ở phiên bản này"